nomad/constants.py

```diff
@@ -1,6 +1,12 @@
 # -*- coding: utf-8 -*-
 
 
+# PROJECTS
+# --
+
+METADATA_BASE_DIR = '.nomad'
```

Comment:
And with the help of `user.nomad.made`, we could easily create a command to purge "orphan" containers because it's still possible to reverse-find (or *not* find) a `nomad.yml` file from a LXD container.